tests/documentation/DocumentationStyleBearTest.py

```diff
@@ -66,6 +66,33 @@ def test_MalformedComment_function(self):
     return test_MalformedComment_function
 
 
+def test_use_spaces(test_data, message, optional_setting=None):
+    def test_use_spaces_function(self):
+        arguments = {'language': 'python', 'docstyle': 'default', 'use_spaces': True}
```

Comment:
@damngamerz this was the hit and trail i made to sen `use_space=True`

Reply:
I see. okay. I see what the problem is :)

Reply:
What is it ;P

Reply:
https://github.com/coala/coala-bears/pull/2236/files#diff-0c99afd0816a2e27be9acf1588eb8cfeR134
See for this to work you need to add `use_spaces`, here to the param's of run()  method and then pass it to `process_documentation`
Also I found one more improvement.
please only define types of params in `process_documentation` like `use_spaces: bool`.
Default values should be in run() method. :+1: This can be in another commit. If you want to solve this up else I will open up a separate issue.

Reply:
I will add a new commit ;)